packages/frontend/src/components/ChartDownload.tsx

```diff
@@ -215,18 +209,14 @@ const ChartDownloadOptions: React.FC<DownloadOptions> = ({
                     ]}
                 />
             )}
-
-            {!isTable && (
-                <Button
-                    size="xs"
-                    ml="auto"
-                    leftIcon={<MantineIcon icon={IconDownload} />}
-                    onClick={onDownload}
-                >
-                    {' '}
-                    Download
-                </Button>
-            )}
+            <Button
```

Comment:
we would never reach this statement. It's moved down below before rendering this component.